test/e2e/scaffold/scaffold.go

```diff
@@ -541,12 +564,19 @@ func (s *Scaffold) FormatNamespaceLabel(label string) string {
 
 var (
 	versionRegex = regexp.MustCompile(`apiVersion: apisix.apache.org/v.*?\n`)
+	kindRegex    = regexp.MustCompile(`kind: .*?\n`)
 )
 
 func (s *Scaffold) replaceApiVersion(yml, ver string) string {
 	return versionRegex.ReplaceAllString(yml, "apiVersion: "+ver+"\n")
 }
 
+func (s *Scaffold) getKindValue(yml string) string {
+	kind := strings.Replace(kindRegex.FindString(yml), "\n", "", -1)
```

Comment:
Why not use capture groups?

Reply:
Didn't notice.

Reply:
How about a fix in a follow-up PR? @AlinsRan

Reply:
> How about a fix in a follow-up PR? @AlinsRan

Sure.